test/test_base_variable_importance.py

```diff
@@ -0,0 +1,246 @@
+import numpy as np
+import pytest
+
+from hidimstat.base_variable_importance import BaseVariableImportance
+
+
+@pytest.fixture
+def _set_variable_importance(seed):
```

Comment:
Could we make the name even more explicit, like `set_100_variable_sorted`, so that it's even clearer in the tests how the thresholds are chosen?

Reply:
done